src/main/java/appeng/integration/modules/helpers/MinefactoryReloadedDeepStorageUnit.java

```diff
@@ -36,8 +36,8 @@
 public class MinefactoryReloadedDeepStorageUnit implements IMEInventory<IAEItemStack>
 {
 
-	final IDeepStorageUnit dsu;
-	final TileEntity te;
+	private final IDeepStorageUnit dsu;
+	private final TileEntity te;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "te" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)